apache-activemq-mixin/.lint

```diff
@@ -0,0 +1,22 @@
+exclusions:
+  template-job-rule:
+    reason: "Prometheus datasource variable is being named as prometheus_datasource now while linter expects 'datasource'"
+  panel-datasource-rule:
+    reason: "Loki datasource variable is being named as loki_datasource now while linter expects 'datasource'"
+  template-datasource-rule:
+    reason: "Based on new convention we are using variable names prometheus_datasource and loki_datasource where as linter expects 'datasource'"
+  template-instance-rule:
+    reason: "Based on new convention we are using variable names prometheus_datasource and loki_datasource where as linter expects 'datasource'"
+  target-instance-rule:
+    reason: "Instance selector is not needed for this dashboard, but certain panels take use of the instance variable for querying purposes"
+  panel-title-description-rule:
+    reason: "Not required for logs volume"
+    entries:
+      - panel: "Logs volume" 
+  panel-units-rule:
+    reason: "Custom units are used for better user experience in these panels"
+    entries:
```

Comment:
i think this `entries:` line can be deleted unless the linter complains about it